### PR TITLE
demo scripts init

### PR DESCRIPTION
**Pull request by Kiffolisk**
yes

### CODE REVIEW

File: Demo Scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# What is this?
+This folder contains documentation for scripts that come from the game's demo.
```
Comment (thennothinghappened) [inline Demo Scripts/README.md:2]: Are they accessible in the final game? Please give context on that

File: Demo Scripts/scr_introduction_answer_fail.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Returns: N/A
+### Example:
+```gml
+if (jumped == true) // just an example, not real code
+    scr_introduction_answer_fail()
+```
+This function plays a voiceline depending on the times the action has been repeated. It adds to the "reset counter" in the intro and changes the voiceline every time until the last time.
+
+All the voiceline code:
```
Comment (thennothinghappened) [inline Demo Scripts/scr_introduction_answer_fail.md]: Please refer to CONTRIBUTING.md, please *never* include unmodified code from the game, especially large chunks like this. Our jobs are to describe a function's use and usage, not it's contents, modders can check that themselves.

**Review by thennothinghappened — CHANGES_REQUESTED** [review]
Expand on README to explain further what the "demo scripts" are.

Don't include code blocks ripped straight from the source code.

Fix these two and I'll approve your pull request as soon as I can.

**Comment by Kiffolisk** [comment]
ok tried fixing

**Comment by thennothinghappened** [comment]
Very professionally done now, well done!

**Pull request merged**